src/consumer/fetchManager.js

```diff
@@ -47,7 +51,7 @@ const createFetchManager = ({
         workerQueue,
         fetch: async nodeId => {
           validateShouldRebalance()
-          return fetch(nodeId)
+          return await fetch(nodeId)
```

Comment:
NIT: unnecessary change

Reply:
Accidental leftover from checking to make sure what this resolved to 😅